common/src/main/scala/aws/AsyncDynamo.scala

```diff
@@ -21,6 +23,11 @@ object AsyncDynamo {
   def keyBetween(a: String, b: String): Condition = new Condition()
     .withComparisonOperator(ComparisonOperator.BETWEEN)
     .withAttributeValueList(new AttributeValue(a), new AttributeValue(b))
+
+  def apply(region: Regions, credentialsProvider: AWSCredentialsProvider = new InstanceProfileCredentialsProvider()): AsyncDynamo = {
```

Comment:
Can you use DefaultAWSCredentialsProviderChain instead of InstanceProfileCredentialsProvider?